developer_docs/Containerfile

```
@@ -0,0 +1,15 @@
+FROM fedora:39
+
+RUN dnf upgrade -y && \
+    dnf install -y \
+      findutils \
+      gcc \
+      gcc-c++ \
+      make \
+      redhat-rpm-config \
+      ruby-devel \
+      rubygem-asciidoctor && \
+    dnf groupinstall -y development-tools && \
+    gem install sass
```

We don't need `sass` here since we don't do anything with it. I think you can trim it down to:
```suggestion
RUN dnf install -y rubygem-asciidoctor && dnf clean all

---

Also true; applied. The container image is now 121 MiB instead of 1.42 GiB. :tada: